### PR TITLE
Update methods.yaml

### PR DESCRIPTION
**Pull request by douglaseggleton**
# Description

`googlepay_pan_only` is not a valid method in a transaction response - need to confirm that it isn't used in another request.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own changes
- [ ] I have run `yarn test` to make sure my changes pass all linters
- [ ] I have pulled the latest changes from the upstream `main` branch

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

**Comment by gryevns** [comment]
I think this is valid - do we need to add `googlepay_pan_only` [here](https://github.com/gr4vy/gr4vy-openapi/blob/d99159f69c9f5133e0e72ee55ba2a75bfae00707/reference/request-bodies/google_pay_request.yaml#L15-L20)?

File: reference/request-bodies/google_pay_request.yaml [hunk 1]
```diff
@@ -18,6 +18,7 @@ properties:
     example: googlepay
     enum:
       - googlepay
+      - googlepay_pan_only
```
Comment (gryevns) [inline reference/request-bodies/google_pay_request.yaml:21]: Sorry, double checked this and it's not the case, we only need `googlepay` so can revert this line. https://github.com/gr4vy/core-api/blob/429bbff9951d1109a57a488af003e2703dbb54f8/src/api/routers/payment_methods/schemas.py#L268

Comment (thomas-gr4vy) [reply reference/request-bodies/google_pay_request.yaml:21]: As per Gary's comment, `googlepay_pan_only` is not required. It is only used internally to Core API now. 👍

Comment (douglaseggleton) [inline reference/request-bodies/google_pay_request.yaml:21]:
```suggestion
```

**Pull request merged**